montblanc/impl/biro/v2/loaders/loaders.py

```diff
@@ -23,6 +23,10 @@
 import montblanc
 import montblanc.impl.common.loaders
 
+ORDER_CASA = 'casa'
+ORDER_OTHER = 'other'
+VALID_ORDERINGS = [ORDER_CASA, ORDER_OTHER]
```

Comment:
Out of interest, is it the case that **baseline x time** is the default CASA ordering? I originally started coding it this way, but it was changed to **time x baseline** after design discussions.